features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json

```diff
@@ -222,6 +222,9 @@
   "account_recovery.endpoint.auth.name":"dashboard",
   "account_recovery.endpoint.auth.hash":"66cd9688a2ae068244ea01e70f0e230f5623b7fa4cdecb65070a09ec06452262",
 
+  "authorization_control.skip_authorization.client_authentication.name":"ClientAuthentication",
```

Comment:
shouldn't this be authorization_control.skip_authorization.authenticator.name?

Reply:
`authorization_control.skip_authorization` is the common part and next part need to be the authenticator name to support to configure any number of authenticators from xml.j2 template.